lib/native/io.ts

```diff
@@ -15,32 +15,60 @@
 
 import crt_native = require('./binding');
 import { NativeResource } from "./native_resource";
-import { ResourceSafe } from '../common/resource_safety';
-import { InputStreamBase } from '../common/io';
+import { InputStreamBase, TlsVersion } from '../common/io';
 import { Readable } from 'stream';
+export { TlsVersion } from '../common/io';
```

Comment:
double import?

Reply:
JK fuck exports

Reply:
Nope, import and export are separate/different.